x-pack/plugin/ml/src/test/java/org/elasticsearch/xpack/ml/datafeed/extractor/fields/ExtractedFieldTests.java

```diff
@@ -98,13 +96,6 @@ public void testNewTimeFieldGivenSource() {
         expectThrows(IllegalArgumentException.class, () -> ExtractedField.newTimeField("time", ExtractedField.ExtractionMethod.SOURCE));
     }
 
-    public void testValueGivenTimeField() {
-        final long millis = randomLong();
```

Comment:
was this intentionally removed and not updated to `ZonedDateTime`?

Reply:
the only values expected to be returned from searchHit are of types Long and String.
there is a separate testValueGivenStringTimeField but I think it wouldn't do any harm to add another testcase for testing Long. 
The one for ZonedDateTime/DateTime is not needed